packit_service/worker/build/build_helper.py

```diff
@@ -131,7 +134,8 @@ def api(self) -> PackitAPI:
             self._api = PackitAPI(
                 self.service_config,
                 self.job_config,
-                self.local_project,
+                # so that the local_project is evaluated only if needed
+                Proxy(partial(BaseBuildJobHelper.local_project.__get__, self)),  # type: ignore
```

Comment:
It's really easy with the `Proxy` object, nice! Do we want to have this for other occurrences in packit's codebase as well?

Reply:
that would  be nice, will look into it